openapi/components/schemas/GatewayAccountConfig/Flexepin.yaml

```
@@ -19,3 +19,12 @@ allOf:
         required:
           - apiKey
           - apiSecret
+      settings:
+        type: object
+        description: Flexepin settings object.
+        properties:
+          adjustVoucherAmountCurrency:
+            type: boolean
+            description: Specifies whether to adjust amount/currency base on Flexepin's voucher.
```

nitpick: we should avoid using [possessive-apostrophes](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#possessive-apostrophes)
```suggestion
            description: Specifies whether to adjust the amount or currency based on a Flexepin voucher.
```